server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/WorksheetController.java

```diff
@@ -85,6 +86,12 @@ public ListResponse<WorksheetMetaResp> listWorksheets(
         return Responses.list(worksheetServiceFacade.listWorksheets(projectId, req));
     }
 
+    @PostMapping("/worksheets/recent")
```

Comment:
should use PageResponse, should contains limit parameter with default value

Reply:
Okay，I'll make this change.